scanners/android/android.go

```diff
@@ -40,7 +40,18 @@ func (scanner *Scanner) DetectPlatform(searchDir string) (_ bool, err error) {
 		return false, fmt.Errorf("failed to search for build.gradle files, error: %s", err)
 	}
 
-	return len(scanner.ProjectRoots) > 0, err
+	isRecognized := len(scanner.ProjectRoots) > 0
+
+	if !isRecognized {
```

Comment:
This way kotlin projects will be collected only if there was no android projects at all. It is up to the user if he mixes his project base languages. Please collect both android and kotlin based project paths.

Reply:
This way when the scanner asks the user which repos he want to select from, he will be able to select from both type.